src/DssSpell.sol

```diff
@@ -23,67 +23,20 @@ import "dss-exec-lib/DssAction.sol";
 
 import { DssSpellCollateralAction } from "./DssSpellCollateral.sol";
 
-interface TeleportJoinLike {
-    function file(bytes32,bytes32,address) external;
-    function file(bytes32,bytes32,uint256) external;
-}
-
-interface TeleportRouterLike {
-    function file(bytes32,bytes32,address) external;
-}
-
-interface TeleportFeeLike {
-    function fee() external view returns (uint256);
-    function ttl() external view returns (uint256);
-}
-
-interface EscrowLike {
-    function approve(address,address,uint256) external;
-}
-
-interface TeleportBridgeLike {
-    function l1Escrow() external view returns (address);
-    function l1TeleportRouter() external view returns (address);
-    function l1Token() external view returns (address);
-}
-
-interface StarknetTeleportBridgeLike is TeleportBridgeLike {
-    function l2TeleportGateway() external view returns (uint256); // uniquely returning uint256 on starknet
-    function starkNet() external view returns (address);
-}
-
 contract DssSpellAction is DssAction, DssSpellCollateralAction {
     // Provides a descriptive tag for bot consumption
     // This should be modified weekly to provide a summary of the actions
-    // Hash: cast keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/0d5af5697be495ae7064a4067800d26072c49584/governance/votes/Executive%20vote%20-%20November%209%2C%202022.md -q -O - 2>/dev/null)"
+    // Hash: cast keccak -- "$(wget https://raw.githubusercontent.com/patrick-j-govalpha/community/172ffd2b2d87db1fd0e40b2eeae4b45f1dcb8964/governance/votes/Executive%20vote%20-%20November%2012%2C%202022.md -q -O - 2>/dev/null)"
```

Comment:
needs to be: https://raw.githubusercontent.com/makerdao/community/172ffd2b2d87db1fd0e40b2eeae4b45f1dcb8964/governance/votes/Executive%20vote%20-%20November%2012%2C%202022.md